src/App.js

```diff
@@ -1,16 +1,232 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
+import ChatLog from './components/ChatLog';
 import chatMessages from './data/messages.json';
 
+
 const App = () => {
+  const [chatMessages, setChatMessages] = useState(JSON.parse(`[{
```

Comment:
I suspect that naming the state variable here `chatMessages` caused issues with accessing the `chatMessages` import from line 4, resulting in the copy/paste of the JSON content below.  The import is a global variable, and creating a local variable with the same name has the effect of causing any attempt to access the variable inside this function to be directed to the local variable.  Renaming the local variable would fix this issue:
```suggestion
  const [messages, setChatMessages] = useState(JSON.parse(`[{
```

If you were attempting to create a deep copy of the import, I recommend doing that programmatically rather than with a copy/paste.  One way to do that is with a map statement, and a spread operator:
```
const copyMessages = chatMessages.map((message) => { return {...message} });
```